src/lib/core/errinj.h

```diff
@@ -192,6 +193,12 @@ void errinj_set_with_environment_vars(void);
 		if (errinj(ID, ERRINJ_BOOL)->bparam) \
 			CODE; \
 	} while (0)
+#  define ERROR_INJECT_COND(ID, TYPE, COND, CODE) \
+	do { \
+		struct errinj *inj = errinj(ID, TYPE); \
+		if (COND) \
+			CODE; \
```

Comment:
This won't work when CODE is not a single statement. Please, wrap it into `{}`.

Reply:
Done.